pycon/settings.py

```diff
@@ -600,6 +600,20 @@
     ('pydata', _('PyData')),
 )
 
+CONFERENCE_TALK_DOMAIN = (
```

Comment:
Could we get it as `Choices` (from model-utils) – so that way we can use it easily as enum later?

Similar to this -> https://github.com/EuroPython/epcon/blob/ep2018/conference/fares.py#L17 (but no need for three values, two should be fine).